build.gradle

```diff
@@ -45,6 +45,10 @@ def addCommonConfigurationForJavaModules(Project project) {
 }
 
 def addCommonConfigurationForAndroidModules(Project project) {
+    project.dependencies {
+        provided files("lib/rt-stub.jar")
```

Comment:
Ah right I missed that this was just the build.gradle file for one of the sample projects.